share/completions/s3cmd.fish

```diff
@@ -0,0 +1,121 @@
+# s3cmd
+
+function __s3cmd_is_remote_path
+	commandline -ct | string match -r -- "^s3://.*"
+end
+
+function __s3cmd_is_valid_remote_path
+	commandline -ct | string match -r -- "^s3://.*?/.*"
+end
+
+# Completions to allow for autocomplete of remote paths
+complete -c s3cmd -f -n "__s3cmd_is_valid_remote_path" -a "(s3cmd ls (commandline -ct) 2>/dev/null)"
+complete -c s3cmd -f -n "__s3cmd_is_remote_path"
+
+# Autogenerated from man page /usr/share/man/man1/s3cmd.1.gz
+# Generated against s3cmd version 2.0
+complete -c s3cmd -s h -l help --description 'show this help message and exit.'
```

Comment:
I'd like it if you could clean these descriptions up a bit.

Make them short and to the point, since space is at a premium in this context. Capitalize the first letter consistently, and remove trailing spaces.

Here, that means `Show help and exit`.